src/utils/markdown/index.ts

```diff
@@ -108,6 +109,7 @@ export function createRehypePlugins(config: MarkdownConfig): RehypePlugin[] {
 					rehypeUnicornElementMap,
 					rehypeTwoslashTabindex,
 					rehypeFileTree,
+					rehypeExpandDetailsAndSummary,
```

Comment:
Let's move this to the EPUB format, remove the `[WIP]` title, then we can merge

Reply:
It's in HTML format for a testing purposes. Unfortunately when it's moved to EPUB format it's not working as expected.

I tested it on macOS, ipadOS and on Kindle ebook reader. In those cases <details> elements are not expanded, even though this rehype plugin works correctly on HTML format.

Reply:
I can't find a solution for this behaviour.

Reply:
As discussed, we need to transform those elements into `divs` and add a new class that will style them to look like `<details>` elements